### PR TITLE
feat-modal

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (90)

### CODE REVIEW

**Review by nishasen — APPROVED** [review]
Code looks clean and understandable. Just a suggestion, you can add 1 or 2 more variations.

File: stylesheets/components/modal.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+
+.basic-modal{
+     height: 6rem;
+     width: 25rem;
+     border: 0.1rem solid var(--black);
+     margin: 5rem;
+}
+.modal-text{
+     font-size: 13px;
```
Comment (RahulSingh9131) [inline stylesheets/components/modal.css]: use rem here too.

**Review by Sameony — APPROVED** [review]
Very concise and clean code, however, instead of using margin to position the elements, you can use the flex-end property of justify content. Do look it up later.

**Review by RahulSingh9131 — APPROVED** [review]
make atleast two variants. Else everything looks good to me.

File: stylesheets/components/modal.css (hunk 1 above)
Comment (iamsaurav28) [reply stylesheets/components/modal.css]: okay done

**Pull request merged**